hms-api/src/service/idea-service.ts

```diff
@@ -147,25 +142,17 @@ export async function editIdea(
 
 export async function getIdeaResponse(id: Uuid): Promise<IdeaResponse> {
   const idea = await getIdea(id);
-
-  let ownerParticipant;
-  try {
-    ownerParticipant = await getParticipant(idea.ownerId);
```

Comment:
there are also calls to `getParticipants` in this function which I imagine should also not exist right? isn't the idea to remove the concept of a participant completely?

Reply:
or did you wanna do that in multiple parts?

Reply:
for hackathons itself I would like to continue with the concept of idea participants 
I just wanted to remove the idea of the owner to be a idea participant, as the owner should be a user

Reply:
hmmm I would say then this gets a bit confusing honestly.  Better to be consistent and just use 1 concept for everything

Reply:
to be honest this would be a lot of work to also replace idea participants with users so this will not be part of this PR 
probably worth a new big ticket

Reply:
then for sure make another to do that as well